### PR TITLE
[#121] Enforcing DB Rules

### PR DESCRIPTION
**Pull request by Jbcampbe**
ticket: #121

### CODE REVIEW

File: app/routes/add.js [hunk 1]
```diff
@@ -16,206 +16,120 @@ export default Ember.Route.extend(AuthenticatedRouteMixin, {
...
+                        required: true
+                    },
+
+                    brand: {
+                        required: true
+                    },
+
+                    custombrand: {
+                        required: true,
+                        maxlength: 40
```
Comment (Jbcampbe) [inline app/routes/add.js]: I thought the db max length was 40 so that is what I did for all of them. I can correct as needed

Comment (ekraihan) [inline app/routes/add.js]: I believe,
Serial: 40 chars
Model: 30 chars

I would ask jere for details

**Comment by ekraihan** [comment]
@Jbcampbe so, are you replacing jquery validate in this PR?

File: app/routes/add.js (hunk 1 above)
Comment (Jbcampbe) [inline app/routes/add.js]: @JerebChase would you be able to get me a list of max char counts for the fields in the tool table? Thanks!

**Comment by Jbcampbe** [comment]
@ekraihan any concerns on this merging?

**Comment by ekraihan** [comment]
@Jbcampbe can i merge this?

**Comment by Jbcampbe** [comment]
Yup

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Pull request merged**